code/modules/badges/badges.dm

```diff
@@ -0,0 +1,88 @@
+/*
+
+Loads all badges from the badge_ranks.txt file.
+Format should be
+rank|icon
+Where:
+Rank - The name of the admin rank
+Icon - The name of the icon state in the badges.dmi file
+
+Special ranks:
+- Donator: Matches clients in the donator list
+- Mentor: Matches clients in the mentor list
+
+*/
+
+//Global list of badges
+GLOBAL_LIST_EMPTY(badge_data)
```

Comment:
C tier name

Reply:
It's a list of the data associated with badges, what else can I put for it?